terraform/terraform.tfvars

```diff
@@ -0,0 +1,10 @@
+# Mandatory Config Options
```

Comment:
I think this base module is not intended to be used alone, so we probably don't need this file.

Reply:
Removed